lib/Migrator/APIDiffMigratorPass.cpp

```diff
@@ -605,8 +605,11 @@ struct APIDiffMigratorPass : public ASTMigratorPass, public SourceEntityWalker {
                                           Arg->getStartLoc().getAdvancedLoc(1));
 
           // Replace "x.getY(" with "x.Y =".
-          Editor.replace(ReplaceRange, (llvm::Twine(Walker.Result.str().
-                                                   substr(3)) + " = ").str());
+          auto Replacement = (llvm::Twine(Walker.Result.str()
+                                          .substr(3)) + " = ").str();
+          Replacement[0] = tolower(Replacement[0]);
```

Comment:
But IIRC this is what the Clang Importer uses to import names with ObjC styling:
https://github.com/apple/swift/blob/master/lib/ClangImporter/ImportName.cpp#L1454